tasks/system/Linux.yml

```diff
@@ -30,7 +30,6 @@
       state: "directory"
       owner: "{{ sonar_user }}"
       group: "{{ sonar_group }}"
-      mode: "0555"
```

Comment:
why this mode changed while installation ?

Reply:
it's not changed, but sonar user can't unarchive zip into /opt/sonarqube dir that have dr-xr-xr-x mode